recipes/ffmpeg/all/conanfile.py

```diff
@@ -311,14 +314,14 @@ def requirements(self):
             self.requires("openssl/[>=1.1 <4]")
         if self.options.get_safe("with_libalsa"):
             self.requires("libalsa/1.2.10")
-        if self.options.get_safe("with_xcb") or self.options.get_safe("with_vaapi"):
+        if self.options.get_safe("with_xcb"):
             self.requires("xorg/system")
         if self.options.get_safe("with_pulse"):
             self.requires("pulseaudio/14.2")
         if self.options.get_safe("with_vaapi"):
-            self.requires("vaapi/system")
+            self.requires("libva/2.20.0")
```

Comment:
libva looks a good replacement. 👍 

I just compared both packages and they look equivalent. The only possible problem that may cause is breakage for few users, as it is closer to hardware and the package may be incompatible. Of course, rebuilding from source should be enough.

About libva distributed from AUR (Arch Linux):

- https://archlinux.org/packages/extra/x86_64/libva/
- https://gitlab.archlinux.org/archlinux/packaging/packages/libva/-/blob/main/PKGBUILD?ref_type=heads
- https://archlinux.org/packages/extra/x86_64/libva/files/

About libva distributed from APT (Ubuntu):

- https://packages.ubuntu.com/focal/amd64/libva-dev/filelist
- https://git.launchpad.net/ubuntu/+source/libva/tree/debian/control?h=applied/ubuntu/devel

Both packages look be very similar with the Conan libva package, in terms of headers and libraries.